mct_quantizers/pytorch/quantizers/weights_inferable_quantizers/weights_uniform_inferable_quantizer.py

```diff
@@ -12,19 +12,125 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+from typing import Any, List
 
 import numpy as np
 
 from mct_quantizers.common.base_inferable_quantizer import mark_quantizer, QuantizationTarget, QuantizerID
-from mct_quantizers.common.constants import FOUND_TORCH
+from mct_quantizers.common.constants import FOUND_TORCH, FOUND_ONNXRUNTIME_EXTENSIONS
 from mct_quantizers.common.quant_info import QuantizationMethod
+from mct_quantizers.common.quant_utils import adjust_range_to_include_zero
 from mct_quantizers.logger import Logger
 
+if FOUND_ONNXRUNTIME_EXTENSIONS:
+    from onnxruntime_extensions import onnx_op, PyCustomOpDef
+
+
+    def quantize_uniform_weights_numpy(input_tensor: np.ndarray,
+                                       num_bits: int,
+                                       min_range: np.ndarray,
+                                       max_range: np.ndarray,
+                                       per_channel: bool,
+                                       channel_axis: int):
+        """
+           Quantizes the input tensor symmetrically using numpy.
+
```

Comment:
Same in here: replace input tensor with weights tensor,
Check if you can reuse existing quantizers function for weights